src/routes/route.go

```diff
@@ -76,6 +76,8 @@ func New() *echo.Echo {
 	medicalRecord.GET("/patient/id/:id", caHandler.MedicalRecord.ShowMedicalRecordByPatientIDHandler)
 	medicalRecord.GET("/patient/nik/:nik", caHandler.MedicalRecord.ShowMedicalRecordByPatientNIKHandler)
 	medicalRecord.GET("/:id", caHandler.MedicalRecord.ShowMedicalRecordByIDHandler)
+	medicalRecord.PUT("/:id", caHandler.MedicalRecord.AmendMedicalRecordByIDHandler)
+	medicalRecord.DELETE("/:id", caHandler.MedicalRecord.RemoveMedicalRecordByIDHandler)

```

Comment:
please add the doctor middleware to prevent the other role from accessing it

Reply:
Applied https://github.com/ALTERA-CAPSTON-41/backend/pull/68/commits/a1497c5beb6996c51daff09bc6fec44a657cd62b.